hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -106,6 +106,12 @@
   private final OzoneManager impl;
   private OzoneManagerDoubleBuffer ozoneManagerDoubleBuffer;
 
+
+  public OzoneManagerRequestHandler(OzoneManager om) {
+    this.impl = om;
+    this.ozoneManagerDoubleBuffer = null;
```

Comment:
Do we really need this new constructor?  `null` can be passed to the existing one.